Resources/Prototypes/Entities/Objects/Devices/Circuitboards/Machine/production.yml

```diff
@@ -425,9 +425,8 @@
     - type: MachineBoard
       prototype: KitchenReagentGrinder
       requirements:
-        Laser: 1
-        ScanningModule: 1
-        MatterBin: 2
```

Comment:
B-but reagent grinders have storage!
Why can't the storage be upgraded with more matter bins :(

Reply:
imma be real i wasn't even aware they had storage